framework/Steps/IOStep.py

```diff
@@ -118,10 +118,10 @@ def _localInitializeStep(self,inDictionary):
                                                        okay = 'Database',
                                                        received = inDictionary['Output'][i].type))
       # from ROM model to ...
-      elif isinstance(inDictionary['Input'][i], Models.ROM):
+      elif isinstance(inDictionary['Input'][i], Models.ROM) or isinstance(inDictionary['Input'][i], Models.ExternalModel):
```

Comment:
this can be changed to
```python
elif isinstance(inDictionary['Input'][i], (Models.ROM, Models.ExternalModel))
```
right? It would be easier to read and maintain the acceptable types this way.

Reply:
I agree, fixed (and the other two).